### PR TITLE
Add README.md file

### PR DESCRIPTION
**Pull request by duncan-lau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Bluedot Android PointSDK xamarin wrapper
+This repository contains a project to make a C# wrapper for Bluedot Android PointSDK.
+
+## Building
+
+1. Download the latest Android Point SDK AAR Jar file from https://github.com/Bluedot-Innovation/PointSDK-Xamarin-Android/blob/master/Jars/pointsdk-release.aar, and overwrite the previous version of Android SDK in the `Jars` folder.
```
Comment (nehabluedot) [inline README.md:6]: Latest AAR should donwloaded from gitlab release pipeline URL:https://gitlab.com/bluedotio/android/point_sdk_android/-/pipelines

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+2. Open solution, then open project **Options -> NuGet Package -> Metadata** and update version.
+
+3. Build the project with `release` configuration. It will produce `Bluedot.PointSDK.Android.X.Y.Z.nupkg` in `PointSDK-Xamarin-Android/bin/Release/`.
+
+## Validating
+There is a Integration sample at `git@github.com:Bluedot-Innovation/PointSDK-Xamarin-minimal-app-Android.git`. The clone is a primary source of any modifications.
```
Comment (poborin) [inline README.md]:
```suggestion
There is a Integration sample at [git@github.com:Bluedot-Innovation/PointSDK-Xamarin-minimal-app-Android.git](git@github.com:Bluedot-Innovation/PointSDK-Xamarin-minimal-app-Android.git). The clone is a primary source of any modifications.
```

**Review by poborin — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (poborin) [reply README.md]: It will make url clickable.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Bluedot Android PointSDK xamarin wrapper
```
Comment (andrew-stirling) [inline README.md]:
```suggestion
# Bluedot Android PointSDK Xamarin wrapper
```

Comment (duncan-lau) [reply README.md]: Readme updated

File: README.md (hunk 2 above)
Comment (duncan-lau) [reply README.md]: Readme updated

**Review by nehabluedot — APPROVED** [review]

**Pull request merged**